pkgs/applications/misc/azuredatastudio/azuredatastudio.desktop

```diff
@@ -0,0 +1,18 @@
+[Desktop Entry]
```

Comment:
Please use the appropriate function to create this file and place it inline with the default.nix.